cmds/identityd/main.go

```diff
@@ -88,7 +88,7 @@ func main() {
 
 	if farm {
 		env := environment.MustGet()
-		fmt.Println(env.FarmerID)
+		fmt.Println(env.FarmID)
```

Comment:
for that change do we have the relevant docs / bootstrap updated with that change?

Reply:
No, it's only the internal code in zos that changed the name